quiz.sql

```diff
@@ -6,38 +6,41 @@ CREATE DATABASE westeros;
 --   `houses` has the following columns: id (serial primary key), name (text), motto (text), and sigil (text)
 --   `people` has an id ( serial primary key), name (text), title (text), quote (text), image (text), and age (integer)
 
--- your code starts here
+CREATE TABLE houses (id SERIAL PRIMARY KEY, name TEXT, motto TEXT, sigil TEXT);
 
--- your code ends here
+CREATE TABLE people (id SERIAL PRIMARY KEY, name TEXT, title TEXT, quote TEXT, image TEXT, age INTEGER);
 
 -- Question 2. Load the data from `westeros/houses.csv` into the `houses` table and the data from `westeros/people.csv` into the `people` table.
 
--- your code starts here
+\copy houses(name, motto, sigil) FROM './westeros/houses.csv' WITH (FORMAT csv, HEADER true);
 
--- your code ends here
+\copy people(name, title, quote, image, age) FROM './westeros/people.csv' WITH (FORMAT csv, HEADER true);
 
 -- Question 3. Insert `Cersei Baratheon,King Mother and Lady of Casterly Rock,"The gods have no mercy, that’s why they’re gods.",http://i.imgur.com/jYeeCJ3.jpg,40`
 -- into the people table, and `House Baratheon,Ours is the fury.,http://i.imgur.com/iJtBKIR.png`
 -- into the houses table
 
--- your code starts here
+INSERT INTO people(name, title, quote, image) VALUES ('Cersei Baratheon', 'King Mother and Lady of Casterly Rock', '"The gods have no mercy, that’s why they’re gods."', 'http://i.imgur.com/jYeeCJ3.jpg,40');
```

Comment:
Note that the double quotes will be actual characters in the quote string as a result of this `INSERT`. The inside single quotes are actually not single quotes, but fancy unicode single-quotes (character 0x2019), which are not parsed as single quotes and need not be escaped.